CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 # master (unreleased)
 
-Nothing here yet, stay tuned.
+- Run tox with Django 1.11
```

Comment:
please add:

* the PR identifier (#22),
* add an item stating that we're testing against Django master version